helm/values.yaml

```diff
@@ -64,51 +82,165 @@ resources:
       cpu: 100m
       memory: 128Mi
 
+
 nginx:
   enabled: true
+  fullnameOverride: "export-management-export-management-nginx"
   replicaCount: 2
   image:
     repository: nginx-otel-unprivileged
-    tag: 'v1.0.0' 
+    tag: "v1.0.0"
+  envoy:
+    image: envoyproxy/envoy
+    imageTag: v1.20.7
+    port: &envoyPort 8000
+    maxRequestBytes: 8192
+    requestedDomain: rasterExport
+    resources:
+      enabled: true
+      value:
+        limits:
+          cpu: 100m
+          memory: 128Mi
+        requests:
+          cpu: 100m
+          memory: 128Mi
+  opa:
+    image: openpolicyagent/opa
+    imageTag: 0.29.4-envoy-3
+    port: 9191
+    createSecret: true
+    jwtPublicKeySecretName: map-colonies-jwks
+    customHeaderName: "x-api-key"
+    secretName: docker-mapproxy-nginx-opa
+    enableLogging: false
+    logLevel: warn
+    resources:
+      enabled: true
+      value:
+        limits:
+          cpu: 100m
+          memory: 128Mi
+        requests:
+          cpu: 100m
+          memory: 128Mi
+
   port: 8080
   internalServicePort: 80
-  targetPort: 8080
-  servicePort: 8080
-  nodePort: 30001
+  nginxTargetPort: &nginxTargetPort 8080
+  targetPort: *nginxTargetPort  # --> if global.opalaEnabled is true use *nginxTargetPort else use *envoyPort | if no authorization is needed use *nginxTargetPort
+  nodePort: 30003
+
   authorization:
     enabled: true
-    domain: 'rasterExport'
-    url: http://opa-la-opa-service/v1/data/http/authz/decision
+    domain: "rasterExport"
+    url: http://opa-int-opa-service.infra-services/v1/data/http/authz/decision
```

Comment:
for dev

Reply:
this is the url for the opala service for int/prod